content/en/docs/concepts/security/cloud-native-security.md

```diff
@@ -24,7 +24,7 @@ There are localized versions available of this whitepaper; if you can link to on
 when localizing, that's even better.
 {{< /comment >}}
 
-The CNCF [white paper](https://github.com/cncf/tag-security/tree/main/security-whitepaper)
+The CNCF [white paper](https://github.com/cncf/tag-security/tree/main/community/resources/security-whitepaper)
```

Comment:
I think we should link to https://github.com/cncf/tag-security/blob/main/community/resources/security-whitepaper/v2/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf (see https://github.com/cncf/tag-security/tree/main/community/resources/security-whitepaper)

Reply:
@sftim Thanks for the suggestion. I will update the referenced link.